packages/xdg_directories/example/integration_test/xdg_directories_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show Platform;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:xdg_directories/xdg_directories.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('XDG Directories', (WidgetTester _) async {
+    // Check that the test is running on Linux.
+    if (!Platform.isLinux) {
```

Comment:
There are no other platforms supported by the example, so this check should not be necessary.

(If it were, the way to do it would be with the `skip` parameter to `testWidgets`, not an assertion.)

Reply:
Thanks.